### PR TITLE
Use large problem size in the auto-aggregated indexgather

### PR DESCRIPTION
**Pull request by e-kayrakli**
When I created this test, I just used the execopts for the unoptimized version
which is too small for CS. It causes a big difference between the manual and
auto aggregated versions:

https://chapel-lang.org/perf/16-node-cs-hdr/?startdate=2021/01/21&enddate=2021/03/15&graphs=baleaggregatedindexgatherperfmbspernode

This PR makes the auto-aggregated version always use the large problem size.

### CODE REVIEW

File: test/studies/bale/indexgather/ig-auto-agg.ml-execopts [hunk 1]
```diff
@@ -9,9 +9,6 @@ comm_sub = os.getenv('CHPL_COMM_SUBSTRATE')
 ugni = comm == 'ugni'
 gn_aries = comm == 'gasnet' and comm_sub  == 'aries'
```
Comment (ronawho) [inline test/studies/bale/indexgather/ig-auto-agg.ml-execopts:10]: Can remove all the config checking and probably comment above if you want too.

Comment (e-kayrakli) [reply test/studies/bale/indexgather/ig-auto-agg.ml-execopts]: Yep! I just switched to a plain-old-execopts file

File: test/studies/bale/indexgather/ig-auto-agg.ml-execopts [hunk 2]
```diff
@@ -1,17 +1,4 @@
...
-N  = 10000
-NL = 1000000
-if ugni or gn_aries:
-  N = NL
-
-print('--N={0} --printStats --mode=Mode.ordered    # bale-ig-auto-aggregated'.format(N))
+# other versions of this test chooses N based on the comm layer and the version
+# of the benchmark kernel. However, in auto/manual aggregation we always use
+# larger N. See ig.ml-execopts
+--N=1000000 --printStats --mode=Mode.ordered    # bale-ig-auto-aggregated'.format(NL))
```
Comment (ronawho) [inline test/studies/bale/indexgather/ig-auto-agg.ml-execopts:4]: Trailing/leftover `'.format(NL))`

**Review by ronawho — APPROVED** [review]

File: test/studies/bale/indexgather/ig-auto-agg.ml-execopts (hunk 2 above)
Comment (e-kayrakli) [reply test/studies/bale/indexgather/ig-auto-agg.ml-execopts]: I don't know what to say. Thanks for catching it.

**Pull request merged**